test/lib/ansible_test/_data/sanity/code-smell/runtime.py

```diff
@@ -109,15 +109,28 @@ def main():
     list_dict_import_redirection_schema = [{str_type: import_redirection_schema}
                                            for str_type in string_types]
 
+    # action_groups_redirection schema
+
+    action_group_redirection_schema = Any(
+        Schema(*string_types),
```

Comment:
Is this line needed?

Reply:
ah, good spot, not it isn't, and neither is the same line in the other schemas
This incorrectly allowed ` `{redirect: <str>}`, as you said on IRC.